style.css

```diff
@@ -387,6 +407,180 @@ button:active {
+
+/* ------------------------- */
+
+.popup__window {
+  display: none;
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background: #c1c7d0;
+  background-blend-mode: multiply;
+  z-index: 100;
+  overflow-y: auto;
+  overflow-x: hidden;
+}
+
+.popup-content {
+  padding: 2rem;
+  width: 90%;
+  max-width: 920px;
+  max-height: 1083px;
+  margin: 5% auto 2% auto;
+  background: #fff;
+  border: 1px solid #ebecf0;
+  box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
+  border-radius: 8px;
+  z-index: 200;
+}
+
+.popup-content__wrapper {
+  display: grid;
+  grid-template-columns: 80vw;
+  grid-template-areas:
+    "img"
+    "title"
+    "tags"
+    "description"
+    "buttons";
+  justify-content: center;
+  align-items: center;
+}
+
+#popup__close__button {
+  grid-area: 1/1/2/2;
+  align-self: start;
+  justify-self: end;
+  margin: 1rem;
+  width: 40px;
+  height: 40px;
+  cursor: pointer;
+  position: relative;
+}
+
+#popup__close__button .bar {
+  position: absolute;
+  width: 20px;
+  height: 2px;
+  background-color: var(--color-white);
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+#popup__close__button .bar:first-child {
+  transform: translate(-50%, -50%) rotate(45deg);
+}
+
+#popup__close__button .bar:last-child {
+  transform: translate(-50%, -50%) rotate(-45deg);
+}
+
+.popup__picture {
+  grid-area: img;
+}
+
+.popup__picture img {
+  width: 100%;
+  max-width: 910px;
+  height: 100%;
+  border-radius: 8px;
+}
+
+.popup__title {
+  grid-area: title;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 3.2rem;
+  line-height: 4.4rem;
+  text-align: left;
+  color: #172b4d;
+  margin: 2rem 0;
+}
+
+li:nth-child(n + 4) {
+  display: none;
+}
+
+#popup__card__tags {
+  grid-area: tags;
+  justify-content: left;
+  width: 100%;
+}
+
+.about-me__card__tags li {
+  height: 40px;
+  background: #fff;
+  border-radius: 8px;
+  color: #36b37e;
+}
+
+#popup__card__tags li {
+  margin-right: 8px;
+  padding: 0 10px;
+}
+
+.popup__description {
+  grid-area: description;
+  width: 100%;
+  min-width: 0;
+  margin: 2rem 0 1rem 0;
+  text-align: justify;
+  line-height: 140%;
+  font-size: 100%;
+}
+
+.popup__buttons__container {
+  grid-area: buttons;
+  display: flex;
+  flex-flow: row nowrap;
+  justify-content: space-between;
+  align-items: flex-start;
+  align-self: start;
+  margin-top: 1vh;
+  gap: 12px;
+  width: 100%;
+  height: 70%;
+}
+
+#popup__live__link__button {
+  width: 38vw;
+  max-width: 130px;
+  margin: 0;
+  font-size: 100%;
+  padding: 0 4%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+#popup__live__link__button img {
+  width: 18px;
+  margin-left: 1rem;
+}
+
+#popup__source__link__button {
+  width: 44vw;
+  max-width: 150px;
+  margin: 0;
+  font-size: 100%;
+  padding: 0 4%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+#popup__source__link__button img {
+  width: 18px;
+  margin-left: 1rem;
+}
+
```

Comment:
Nice job so far. Kindly ensure that the popup design matches that of the actual design in terms of the padding and margins since it is part of the project requirements. 

- [ ] The distance between the see live and see source buttons increases with increasing widths as shown below. 
- [ ] The close button on the top right corner also seems to appear twice
Kindly check and fix this errors. 

### Your design 


![Screenshot from 2023-05-19 06-16-30](https://github.com/sergio-usma/Personal_portfolio/assets/65233860/e4c603bb-98ff-4c5c-8f8d-b69878878612)


### Actual design 

![Screenshot from 2023-05-19 06-16-48](https://github.com/sergio-usma/Personal_portfolio/assets/65233860/529c428b-240f-4f7a-8189-7cb54f9c85e7)

Reply:
Hi! @jerryowusu regarding this comment: "The close button on the top right corner also seems to appear twice
Kindly check and fix these errors". The Figma's image has that "X," and it's not possible to download without it, so it is a Microverse template error; try to download it, and you will notice that. I could edit it with Photoshop or a similar image processor, but this is not in the project requirements.

Now, about the design, are you sure the viewport width of the "Actual design" (attached screenshot 2) is the same as "your design" (attached screenshot 1). Because I think screenshot 2 is based on a width of 320px, and screenshot 1 with a width of 580px, so obviously, the space between the buttons will increase; unless I use some transformation property to make it bigger (but this is not in the project requirements), so I can't understand you. Do you want I justify it center or what specifically?

Reply:
So, look @jerryowusu the additional "X" is not a bug, nor an unintentional error. Currently, is a template's error, so this is not my fault... If possible I can replace the image with another, but I don't know if that is allowed...
![image](https://github.com/sergio-usma/Personal_portfolio/assets/126805818/19bb86be-2e25-459c-975e-70fee7364285)